### ISSUE TITLE
Integration not posting anything to channel

### ISSUE THREAD

**Issue by canpoyrazoglu** [integrations/slack#1326]
**Describe the bug**
We've just added Slack integration, everything went smoothly. Then I've pushed some commits but nothing is posted to our Slack channel.

**To Reproduce**
1. Add GitHub bot to Slack. Authorize everthing both on Slack and GitHub.
2. Add the integration to a channel: `/github subscribe myorgname/myreponame` (redacted org and repo name here)
3. See confirmation message: :white_check_mark: Subscribed to myorgname/myreponame. This channel will receive notifications for
issues, pulls, commits, releases, deployments
4. Commit and push something to the repo that has been subscribed. In my case there is also a deployment action triggered on this commit.
5. Nothing gets posted on the channel. 

**Expected behavior**
Channel should receive information about the commit and the deployment result

**Additional context**
We've just moved to GitHub, I've done everything GitHub and Slack says step by step, but by no means I have any experience about this integration so forgive if I'm missing something obvious.

**Comment by sanjeev9160** [comment]
@canpoyrazoglu Thanks for opening the issue. I tried reproing the issue but it works fine for us.

Is your organization under SAML? If yes then please run signout command: `/github signout` and do a signin `make sure to do a SAML authorization`. Post that you will also need to run the subscribe command again.

If it doesn't help out then, please share the output of /github debug so that I can debug the issue. 

Thanks.

**Comment by canpoyrazoglu** [comment]
No, we aren't using SAML. However after trying a bit more I've realized that it's about brances. From googling I've found that some people are recommending `commits:*` to subscribe to pushes to all branches. I think that was the case as I was using `commits` (not `commits:*) to subscribe and my branch wasn't the master/main branch. After doing that it started working.

If that is the default behavior maybe documentation can be updated as something like "commit watches only the main branch, if you want to subscribe to pushes to all branches use commits:*".
